public/locales/de/translation.json

```diff
@@ -11,10 +11,10 @@
             "update_password": "Passwort ändern",
             "first_time_message": "Falls dies das erste Mal ist, dass du dich mit diesem Benutzerkonto anmeldest, solltest du zu den Kontoeinstellungen gehen und dein Passwort aktualisieren",
             "hi": "Hi",
-            "intro": "Danke fürs Einloggen. Hier ist die Liste der Funktionen, die angemeldete Benutzer erhalten",
+            "intro": "Danke fürs Einloggen. Hier ist eine Liste der Funktionen, die angemeldete Benutzer erhalten",
```

Comment:
In the English version, @bjarneo recently removed the "here is a list of features" thing. See [here](https://github.com/HemmeligOrg/Hemmelig.app/blob/86918e546054bd861d43d53068d5a1bea5de409b/public/locales/en/translation.json#L15). I guess something like "Angemeldete Benutzer erhalten mehr Funktionen" (approximate attempt, my German is brittle) would be enough.

Reply:
Indeed, with a much neater wording than my gobbledygook!